dvc/repo/experiments/queue/base.py

```diff
@@ -706,3 +708,57 @@ def stash_failed(self, entry: QueueEntry) -> None:
                 entry.stash_rev,
                 message=f"commit: {msg}",
             )
+
+    def _fetch_running_exp(
+        self, rev: str, infofile: str, fetch_refs: bool
+    ) -> Dict[str, Dict]:
+        from dvc.scm import InvalidRemoteSCMRepo
+        from dvc.utils.serialize import load_json
+
+        from ..executor.local import TempDirExecutor
```

Comment:
`BaseExecutor` should not be importing `TempDirExecutor`.

If this is a generic implementation of `_fetch_running_exp` it should be using `self.from_info()` later. If this is actually a tempdir specific implementation, it belongs in `TempDirExecutor`.